setup.py

```diff
@@ -22,7 +22,7 @@
     version=version,
     license="MIT",
     package_data={
```

Comment:
We could move this to the pyproject.toml file under [tool.setuptools.package-data]

Reply:
Added dependencies and datasets to this in the toml